support-frontend/app/controllers/PaperSubscription.scala

```diff
@@ -59,7 +59,7 @@ class PaperSubscription(
     val css = Left(RefPath("paperSubscriptionLandingPage.css"))
     val canonicalLink = Some(buildCanonicalPaperSubscriptionLink())
     val description = stringsConfig.paperLandingDescription
-    val promoCodes = request.queryString.get("promoCode").map(_.toList).getOrElse(Nil) :+ "SEP2512VHD"
+    val promoCodes = request.queryString.get("promoCode").map(_.toList).getOrElse(Nil)
```

Comment:
from https://github.com/guardian/support-frontend/pull/2083/files#r424632480

Reply:
Have checked and this code is expired now
https://memsub-promotions.gutools.co.uk/#/promotion/7d32b942-2ed2-438b-9ab4-2505b02fde2c

It is also referenced here: https://github.com/guardian/support-frontend/blob/0be6eb9285951f9ef585cf543a02d5b4a76212fa/support-frontend/assets/pages/paper-subscription-landing/components/content/helpers.jsx#L43
So you could remove that as well

Reply:
I could assuming it's a simple obvious change, but looking into the history, it seems to go back [a long way](https://github.com/guardian/support-frontend/commit/1dd0e86945052be89b66739f2ae3cdf06452b359#diff-2f03286c4a1c15bc3c731794eb78bf17R37), and there are code changes that would require me to understand the code a bit. (I guess just remove the line completely but it's not 100% trivial)  So I think it's probably better in another PR, although free free to push a commit to here if you think it's simple enough!

Reply:
Ok, happy to leave it. I have a feeling there's quite a bit of dead code here so it might well be better to spend some time understanding and pruning the whole area the next time it needs to change.